app/assets/javascripts/hotwire_spark.js

```diff
@@ -1600,6 +1600,10 @@ var HotwireSpark = (function () {
       document.body.setAttribute("data-hotwire-spark-ready", "");
     },
     async received(message) {
+      if (!window.Turbo) {
+        log(`Turbo not detected, skipping ${message.action}`);
+        return;
```

Comment:
This would hide errors for apps without Turbo and silently not update the page by default. I think a better approach here would be to always log a descriptive error with an explanation on how to avoid the error, e.g. `[hotwire-spark] Tried to replace the page with Turbo, but Turbo is not available on window.Turbo` or sth. similar :)